test/ApiClient.test.ts

```diff
@@ -7,7 +7,7 @@ import { messageApi } from '@xmtp/proto'
 import { sleep } from './helpers'
 import { LocalAuthenticator } from '../src/authn'
 import { PrivateKey } from '../src'
-import { version } from '../package.json'
+import packageJson from '../package.json'
```

Comment:
Curious why this needed to change

Reply:
certain transpilers were giving a warning about using non-default imports of a json file. it still technically works, but just wanted to remove any possible issues, even though they aren't likely.

more info on why here: https://github.com/tc39/proposal-json-modules#why-dont-json-modules-support-named-exports